src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -23,8 +23,8 @@
 }
 PROPORTION_INITIALIZATION_NO_SSN = 0.14
 PROPORTION_NEWBORNS_NO_SSN = 0.10
-PROPORTION_PERSONS_LEAVING_COUNTRY = 0.01
-PROPORTION_HOUSEHOLDS_LEAVING_COUNTRY = 0.01
+PROPORTION_PERSONS_LEAVING_COUNTRY = 0.05
```

Comment:
was there a reason for this change? just testing?

Reply:
Just testing.  I was having almost no sims move out of the country at the lower rate which made testing more annoying so I was just going to leave it for now.